leetcode/generate_md.py

```diff
@@ -1,63 +0,0 @@
-separator = '<!-----solution----->'
```

Comment:
VERIFICATION RESULT : 
 Hello! 
 I am here to help you with making your pr title looks better. Now it is: 
 Delete generate_md.py 
 Mistakes: 
 Dude...it is not even in right format.